### PR TITLE
fixing file paths

### PR DESCRIPTION
**Pull request by mglavey**
### Checklist For Submitter

* [ ] I have documented all new functionality
* [ ] I have screenshotted new UI changes and attached them to this PR

### Checklist For Reviewer
* [ ] Code formatting/static analysis: is the code formatted properly? Does a linter/other form of static analysis reveal any issues? Does the code have appropriately low cyclomatic complexity?
* [ ] Code architecture: concerns are separated, code follows single-responsibility principle, code utilizes OOP, DRY code.
* [ ] Coding best practices: code avoids hard-coded variables, inefficient computations, reinventing the wheel when using frameworks. Code is well-commented and generally readable.
* [ ] Non-functional requirements: Code has accompanying tests (if using TDD). Programmer can easily explain decisions to reviewers.

### CODE REVIEW

**Comment by mglavey** [comment]
Hi Bryan,

Changed the one file path issue. App is compiling just fine now. Not sure why CircleICi check is not working.

File: frontend/src/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import SignUp from './components/SignUp/SignUp.js';
 import UsersAll from './components/AdminControls/UsersAll.js';
 import User from './components/AdminControls/User.js';
 import history from './history.js';
-import RecipeList from './components/recipeCard/recipeCard.js';
+import RecipeList from './components/RecipeCard/recipeCard.js';
```
Comment (vhoof) [inline frontend/src/App.js]: I think the recipeCard.js should have a capital R?

Comment (mglavey) [reply frontend/src/App.js]: The app is compiling just fine and I've triple checked all the naming/capitalization so maybe best to ignore circleCi check?

Comment (EvanGrillo) [reply frontend/src/App.js]: changing line 9 to ```import RecipeList from './components/recipeCard/recipeCard.js';``` should resolve the circleCi issue

File: frontend/src/App.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import SignUp from './components/SignUp/SignUp.js';
 import UsersAll from './components/AdminControls/UsersAll.js';
 import User from './components/AdminControls/User.js';
 import history from './history.js';
-import RecipeList from './components/recipeCard/recipeCard.js';
+import RecipeList from './components/RecipeCard/RecipeCard.js';
```
Comment (EvanGrillo) [inline frontend/src/App.js]: @mglavey in order for this to work, you would need to change the component folder name to 'RecipeCard' & the js file to 'RecipeCard.js' 

this type of minor syntax stuff drives me pretty crazy sometimes

Comment (mglavey) [reply frontend/src/App.js]: Yep, that's what I've been working on. Unfortunately it won't let me commit those changes so I was working to change the folder name manually on github.

Comment (mglavey) [reply frontend/src/App.js]: https://stackoverflow.com/questions/31861651/how-to-rename-a-directory-folder-in-githubweb

Comment (EvanGrillo) [reply frontend/src/App.js]: try 
```git mv recipeCard tmp```
```git mv tmp RecipeCard```

from https://stackoverflow.com/questions/11183788/in-a-git-repository-how-to-properly-rename-a-directory

**Pull request merged**